libs/dataset-explorer/src/lib/DefaultTab.tsx

```diff
@@ -0,0 +1,389 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  AxisConfigDialog,
+  JointDataset,
+  ColumnCategories,
+  cohortKey,
+  ChartTypes,
+  IGenericChartProps,
+  ISelectorConfig,
+  MissingParametersPlaceholder,
+  defaultModelAssessmentContext,
+  ModelAssessmentContext,
+  rowErrorSize
+} from "@responsible-ai/core-ui";
+import { localization } from "@responsible-ai/localization";
+import { AccessibleChart } from "@responsible-ai/mlchartlib";
+import _ from "lodash";
+import {
+  getTheme,
+  IDropdownOption,
+  Dropdown,
+  DefaultButton,
+  Text,
+  IChoiceGroupOption
+} from "office-ui-fabric-react";
+import React from "react";
+
+import { defaultTabStyles } from "./DefaultTab.styles";
+import { generatePlotlyProps } from "./generatePlotlyProps";
+import { SidePanel } from "./SidePanel";
+
+export class IDefaultTabProps {}
+
+export interface IDefaultTabState {
+  xDialogOpen: boolean;
+  yDialogOpen: boolean;
+  colorDialogOpen: boolean;
+  selectedCohortIndex: number;
+  chartProps?: IGenericChartProps;
+}
+
+export class DefaultTab extends React.Component<
```

Comment:
Can we find a more descriptive name? I have no clue what to expect from `DefaultTab`.